_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -155,6 +155,20 @@ kubectl create clusterrolebinding cluster-admin-binding \
     --user=$(gcloud config get-value core/account)
 ```
 
+#### IBM Cloud Container Service (IKS)
+
+Kubernetes 1.9 is generallly available on IBM Cloud Container Service (IKS).  At the time of writing it is not the default version, so to create a new lite cluster:
+
+```bash
+bx cs cluster-create --name my_cluster --kube-version 1.9.3
```

Comment:
@linsun just double confirm, so here the command will create a cluster with admission controller plugin as `MutatingAdmissionWebhook,ValidatingAdmissionWebhook`?

Reply:
@gyliu513 yes, they are turned on by default in our 1.9.x clusters in IKS.

Reply:
Is the certificate manager controller also enabled by default? See the minikube setup steps for specific flags. This will be needed if/when we use k8s CA for cert provisioning.

Reply:
yes - i think so as I didn't have to enable it.